recipes/logic1/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  version: 0.1.0
+
+package:
+  name: logic1
+  version: ${{ version }}
+
+source:
```

Comment:
I tried that, but I believe it's not possible since the project uses [`setuptools_scm`](https://setuptools-scm.readthedocs.io/) for versioning. This means that when run outside of a Git repository, `pip` will not be able to compute a version number.

Reply:
See: https://conda-forge.org/docs/maintainer/knowledge_base/#using-setuptools_scm

Reply:
You'll probably have to use:

```
script_env:
    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
script: ...
```

Reply:
Actually since you're using `recipe.yaml` I think you'll have to do it a little differently. It's not well documented, but it looks like you can expand the `script` section to set env variables and then add a `content` section:
https://github.com/prefix-dev/rattler-build/blob/26309bba6712d26822569d631140b9bb8c5c962c/test-data/recipes/variant_config/recipe.yaml#L9

Reply:
Thanks for the hint. Done.